PrayerTimes.Test/PrayerTimesTests.cs

```diff
@@ -81,5 +81,15 @@ public void TestTimesForYear()
             Assert.Equal(new DateTime(2015, 2, 3), times[33].Date);
             Assert.Equal(new TimeSpan(6, 8, 0), times[33].Fajr);
         }
+
+        [Fact]
+        public void TestGetFajrAngle()
```

Comment:
I'm not very familiar XUnit so maybe I'm missing something, but why doesn't this unit test assert anything?